tests/Microsoft.Identity.Test.Unit/PublicApiTests/ClientCredentialWithRegionTests.cs

```diff
@@ -329,6 +329,89 @@ public async Task MsalForceRegionIsSet_RegionIsUsed()
             }
         }
 
+        [TestMethod]
+        public async Task MsalDisableRegionEnvVariable_API()
+        {
+            using (new EnvVariableContext())
+            using (var harness = base.CreateTestHarness())
+            {
+                Environment.SetEnvironmentVariable(
+                    ConfidentialClientApplicationBuilder.DisableRegionEnvVariable, "1");
```

Comment:
"1" is a magic string. Can you define it as a const? Is it used in other tests and already defined as a const?

Reply:
It's not a magic string, the env variable can be set to anything.